README.md

```diff
@@ -25,8 +25,10 @@ git submodule update --init
 #export CFLAGS='-g -O2 -w' 
 #export CXXFLAGS='-g -O2 -w'
 
+mkdir -p cmake/build
+cd cmake/build
+cmake ../..
 make
-sudo make install
```

Comment:
don't we still need to install the package?

Reply:
If google isn't hosting a "latest version" api, i'd recommend switching to the github api instead of hard coding, although it's a bit more verbose:

```
curl   -H "Accept: application/vnd.github.v3+json"   https://api.github.com/repos/grpc/grpc/releases/latest | jq -r .tag_name
```

Maybe pull it into a shell variable to keep the code more linear.

Reply:
Yup! this is reasonable.
I updated the files for more linear code in order to avoid hard coding the latest version.